maxim/tests/test_llamaindex.py

```
@@ -0,0 +1,128 @@
+
+        agent_workflow = AgentWorkflow(
+            agents=[research_agent, write_agent, review_agent],
+            root_agent=research_agent.name,
+            initial_state={
+                "research_notes": {},
+                "report_content": "Not written yet.",
+                "review": "Review required.",
+            },
+        )
+        
+        async def run_workflow():
```

Ruff (0.12.2)

Missing return type annotation for private function `run_workflow`

Add return type annotation: `None`

(ANN202)